be/src/cloud/cloud_meta_mgr.cpp

```diff
@@ -139,11 +139,75 @@ class MetaServiceProxy {
 public:
     static Status get_client(std::shared_ptr<MetaService_Stub>* stub) {
         TEST_SYNC_POINT_RETURN_WITH_VALUE("MetaServiceProxy::get_client", Status::OK(), stub);
-        return get_pooled_client(stub);
+        return get_pooled_client(stub, nullptr);
+    }
+
+    static Status get_proxy(MetaServiceProxy** proxy) {
+        std::shared_ptr<MetaService_Stub> stub;
```

Comment:
add comment for this function, and the stub is a placeholder

Reply:
done